cypress/support/elements/clue/cCanvas.js

```diff
@@ -358,6 +358,32 @@ class ClueCanvas {
         dialog.getDialogOKButton().click();
         dialog.getDialogTitle().should('not.exist');
     }
+
+    // Tile toolbars are in portals at the document 'body' level.
+    // These methods allow working with toolbar buttons even when invoked from,
+    // say, a `within` clause scoped to a tile.
```

Comment:
I added comments.
Also added a `length==1` assertion so that it should complain if there are ever more than one when you go to click a toolbar button.

Reply:
looks good.